hyx/ratelimit/managers.py

```diff
@@ -16,59 +15,21 @@ class TokenBucketLimiter(RateLimiter):
     Otherwise, it's going to be rejected with RateLimitExceeded
     """
 
-    __slots__ = (
-        "_max_executions",
-        "_per_time_secs",
-        "_bucket_size",
-        "_loop",
-        "_token_per_secs",
-        "_tokens",
-        "_next_replenish_at",
-    )
+    __slots__ = ("_token_bucket",)
 
     def __init__(self, max_executions: float, per_time_secs: float, bucket_size: Optional[float] = None) -> None:
-        self._max_executions = max_executions
-        self._per_time_secs = per_time_secs
-
-        self._bucket_size = bucket_size if bucket_size else max_executions
-
-        self._loop = asyncio.get_running_loop()
-        self._token_per_secs = self._per_time_secs / self._max_executions
-
-        self._tokens = self._bucket_size
-        self._next_replenish_at = self._loop.time() + self._token_per_secs
+        self._token_bucket = TokenBucket(max_executions, per_time_secs, bucket_size)
 
     @property
     def tokens(self) -> float:
```

Comment:
Maybe in this case, it's better just to expose the bucket (and it will provide the remaining information around token state).

Reply:
Should we just have one property for just token_bucket and remove the existing two properties?

Reply:
> Should we just have one property for just token_bucket and remove the existing two properties?

Yes, that! I would call the property just `bucket`